book/src/dev/develop.md

```diff
@@ -1,5 +1,13 @@
 # Developer's Guide
 
+## Backtraces
+
+You can use the usual `RUST_BACKTRACE=1` environment variable to enable backtraces, if you build and install `flux` as:
```

Comment:
You also get backtraces for `release`, so this is more like to get better backtraces use `dev`

Reply:
good point will update comment